runtimes/peregrine/src/migrations.rs

```diff
@@ -0,0 +1,19 @@
+// KILT Blockchain – https://botlabs.org
```

Comment:
W: Since we won't have that many migrations i wouldn't move that into another module. A migration module should be enough. This will be removed rather sooner than later...